stow/functions.php

```diff
@@ -52,47 +63,25 @@ function stow_setup() {
 				),
 			)
 		);
-
-		// Add child theme editor color pallete to match Sass-map variables in `_config-child-theme-deep.scss`.
-		add_theme_support(
-			'editor-color-palette',
-			array(
-				array(
-					'name'  => __( 'Primary', 'stow' ),
-					'slug'  => 'primary',
-					'color' => '#404040', 
-				),
-				array(
-					'name'  => __( 'Secondary', 'stow' ),
-					'slug'  => 'secondary',
-					'color' => '#f25f70', 
-				),
-				array(
-					'name'  => __( 'Dark Gray', 'stow' ),
-					'slug'  => 'foreground-dark',
-					'color' => '#111111',
-				),
-				array(
-					'name'  => __( 'Gray', 'stow' ),
-					'slug'  => 'foreground',
-					'color' => '#444444',
-				),
-				array(
-					'name'  => __( 'Light Gray', 'stow' ),
-					'slug'  => 'foreground-light',
-					'color' => '#767676',
-				),
-				array(
-					'name'  => __( 'White Smoke', 'stow' ),
-					'slug'  => 'background',
-					'color' => '#f0f0f0',
-				),
-			)
-		);
 	}
 endif;
 add_action( 'after_setup_theme', 'stow_setup', 12 );
 
+function stow_editor_color_palette( $palette ) {
+	$palette[] = array(
+		'name'  => __( 'Dark Gray', 'stow' ),
+		'slug'  => 'foreground-dark',
+		'color' => '#111111',
+	);
+	$palette[] = array(
+		'name'  => __( 'Light Gray', 'stow' ),
+		'slug'  => 'foreground-light',
+		'color' => '#767676',
+	);
+	return $palette;
+}
+add_filter( 'varia_editor_color_palette', 'stow_editor_color_palette' );
+
```

Comment:
@Automattic/theam we need to add this to the themes that have Foreground light/dark options on their editor color palette. It's not the case for all of them. Thanks for this Ben!